index.html

```diff
@@ -990,32 +656,24 @@ <h2>
     Examples
   </h2>
   <pre class="example js" title="Adjusting the number of video feeds based on CPU usage">
```

Comment:
Another example where `unobserve()` is used would be useful as well. `disconnect()` seems to be the same with only one observer. Then an example with multiple observers would be nice.

Reply:
Sure, maybe you could file an issue to add these examples. Would be nice task for @arskama

Reply:
https://github.com/WICG/compute-pressure/issues/43